.gitignore

```diff
@@ -1,19 +1,16 @@
-**/coverage
 **/node_modules
 **/.eslintcache
-**/.nyc_output
 **/.wireit
 **/*.cpuprofile
 **/*.tsbuildinfo
 
+/.helix/languages.toml
 /.history
 /.vscode/settings.json
-/.mocharc.local.jsonc
 /.idea
 
 /packages/*/lib
 /packages/*/out
 /packages/*/dist
-/packages/*/test-out
```

Comment:
I think it's a good idea to keep all the old lines in here for a little bit, to avoid problems when switching branches. For example when I checked out this PR I had 266 pending changes from the `.nyc_output` and `test-out` folders

Reply:
Good call!